zambialii/templates/peachjam/about.html

```diff
@@ -121,6 +121,16 @@ <h2 class="mb-4" id="contact">{% trans 'Contact us' %}</h2>
         </address>
       {% endblocktrans %}
       </p>
+      {% if PEACHJAM_SETTINGS.contact_form_url %}
```

Comment:
Can you move this into peachjam's about.html, so that it just works on all sites? You may need to introduce a new block into the base template to put it into. So lawlibrary, zambialii and malawilii would no longer need special treatment.